package.json

```diff
@@ -34,14 +34,15 @@
     "playcanvas": "^1.32.4",
     "style-loader": "^1.2.1",
     "webpack": "^4.44.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-conditional-loader": "^1.0.12"
+    "webpack-cli": "^3.3.12"
   },
   "scripts": {
     "lint": "eslint --ignore-pattern 'src/lib' --ext .js src",
     "serve": "serve dist",
     "build": "webpack --config webpack.config.js",
     "build:watch": "webpack --config webpack.config.js --watch",
```

Comment:
I don't care about the version of playcanvas-extras, so how about this:
```
__PLAYCANVAS_PATH__: JSON.stringify(process.env.ENGINE_PATH ? path.resolve(__dirname, process.env.ENGINE_PATH) : 'playcanvas'),
__PLAYCANVAS_EXTRAS_PATH__: JSON.stringify('playcanvas/build/playcanvas-extras.js')
```

With this in place, we can invoke the usual watch like this:
`ENGINE_PATH=../engine/build/playcanvas.js npm run build:watch`

Reply:
Updated the PR with this 👍